### PR TITLE
Add createVersion getServices

### PR DESCRIPTION
**Pull request by richard-still-ft**
Added two new methods to retuen the services and to create a new version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "name": "fastly",
   "description": "Fastly API client for Node.js",
   "keywords": "fastly, api, client",
-  "version": "2.1.0",
+  "version": "2.4.0",
```
Comment (commuterjoy) [inline package.json:13]: 2.2.0 ?

Comment (richard-still-ft) [inline package.json:13]: Well we already pull in 2.3.0, I think this file is out of sync with the branches..

**Comment by commuterjoy** [comment]
:+1: aside from the version number

**Comment by commuterjoy** [comment]
@richard-still-ft also, can you change the pull request to merge in to Financial-Times:master (not the commuterjoy:master)

**Pull request closed without merging**